querybook/webapp/components/BoardExpandableSection/BoardExpandableHeader.tsx

```diff
@@ -0,0 +1,103 @@
+import * as React from 'react';
+import clsx from 'clsx';
+import { useDrag } from 'react-dnd';
+
+import history from 'lib/router-history';
+import { getWithinEnvUrl } from 'lib/utils/query-string';
+import { IBoardWithItemIds } from 'const/board';
+
+import { BoardDraggableType } from 'components/DataDocNavigator/navigatorConst';
+
+import { IconButton } from 'ui/Button/IconButton';
+import { UrlContextMenu } from 'ui/ContextMenu/UrlContextMenu';
+import { ListLink } from 'ui/Link/ListLink';
+import { Title } from 'ui/Title/Title';
+
+interface IProps {
+    boardId: number;
+    boardName: string;
+    collapsed: boolean;
+    toggleCollapsed: () => void;
+    boardData?: IBoardWithItemIds;
```

Comment:
boardId, boardName can be inferred from boardData

Reply:
not for public list